packages/ifpack2/example/BlockTriDi.cpp

```diff
@@ -459,8 +498,8 @@ main (int argc, char* argv[])
   TEUCHOS_STANDARD_CATCH_STATEMENTS(verbose, std::cerr, success);
   */
 
-  
-  std::cout<<"End Result: TEST PASSED"<<std::endl;
+  comm->barrier();
```

Comment:
This stops the "TEST PASSED" text being printed 4 times, interleaved randomly with the timer output.